config/STAT_2760.yaml

```diff
@@ -226,14 +226,70 @@ hadron_correlations:
   pi0_hadron_alice:
     pt_trig: [8., 16.]
     centrality: [[0,10]]
+
+  v2_atlas:
+    four: # four-particle cumulants
```

Comment:
Could you please put the HEPdata and/or paper link for the analysis in a comment?